implementations/rust/ockam/ockam_command/src/relay/list.rs

```diff
@@ -18,17 +18,21 @@ use crate::util::node_rpc;
 use crate::{docs, CommandGlobalOpts};
 
 const PREVIEW_TAG: &str = include_str!("../static/preview_tag.txt");
+const LONG_ABOUT: &str = include_str!("./static/list/long_about.txt");
 const AFTER_LONG_HELP: &str = include_str!("./static/list/after_long_help.txt");
 
 /// List Relays
+///
+/// List Relays on your default node. If you pass '--to <NODE>' then it lists the Relays at the given node
```

Comment:
I think these two lines should be deleted right? This piece of text is the one loaded in the `long_about = docs::about(LONG_ABOUT),` line.